easybuild/easyconfigs/o/OpenMolcas/OpenMolcas-22.10-intel-2022a.eb

```diff
@@ -13,31 +16,22 @@ source_urls = ['https://gitlab.com/Molcas/OpenMolcas/-/archive/v%(version)s/']
 sources = ["OpenMolcas-v%(version)s.tar.gz"]
 patches = ['OpenMolcas-21.06_mcpdft_deps.patch']
 checksums = [
-    {'OpenMolcas-v22.10.tar.gz': 'd1d3d6c9bd69486c36f5285a51ac18e75c03540c5b33751855e94abc764d9658'},
-    {'OpenMolcas-21.06_mcpdft_deps.patch': '708763e060aa0174bbe9c0cc336b509fbf492066465908dd919452836a7a793a'},
+    'd1d3d6c9bd69486c36f5285a51ac18e75c03540c5b33751855e94abc764d9658',  # OpenMolcas-v22.10.tar.gz
+    '708763e060aa0174bbe9c0cc336b509fbf492066465908dd919452836a7a793a',  # OpenMolcas-21.06_mcpdft_deps.patch
 ]
```

Comment:
Please do not revert the checksums to the old format.

Reply:
I created the EC file before I was aware it already existed, that is where this, and the other issues, came in. 
I pulled the EC from the already merged PR and added the line with the checksums. I should have done that initially though. :-) 
I hope that addresses all the issues you mentioned. Thanks for your help!

Reply:
thanks for the update!